src/dmd/backend/cgobj.d

```diff
@@ -403,7 +403,7 @@ __gshared
 {
 public seg_data **SegData;
 
-    int seg_count;
+    int seg_length;
```

Comment:
Make `uint` if never negative? More informative and potentially more efficient (division, remainder, bounds checks are cheaper).

Reply:
It'll become a size_t later. Many more PRs to come on this!